docs/howto/installing-local-extensions.md

```diff
@@ -80,21 +125,70 @@ paths:
     themebase: public/theme
     files: public/files
     view: public/bolt-public/view
+
 extensions:
-    - MyextOne\LocalExtensionOne
-    - MyextTwo\LocalExtensionTwo
-    - MyextThree\LocalExtensionThree
+    - Myextension\LocalExtensionOne
+    - Listing\ListingExtension
 ```
 
-Note that each class defined as above eg: `MyextOne\LocalExtensionOne` must have been
-setup using one of the methods defined above.
+Note that each class defined as above eg: `MyextOne\LocalExtensionOne` must
+have been setup using one of the methods defined above.
+
+If you use a PHP autoloader instead of the yaml style one above then you can
+add your extensions in PHP code in either `.bolt.php` or in your own bootstrap.
+
+Example in `.bolt.php`:
 
-If you use a php autoloader instead of the yaml style one above then you can add
-your extensions in PHP code. For each extension to load call the following:
 ```php
-// inside bolt.php
+<?php
+// inside .bolt.php
+
+return [
+    'paths' => [
+        …
+    ]
+    'extensions' => [
+        new MyExtension\LocalExtensionOne(),
+        new Listing\ListingExtension()
+    ]
+];
+```
+
+Example in your own bootstrap:
```

Comment:
Well if you do a custom bootstrap and don't use `.bolt.php`., then `vendor/bin/nut` won't work. If people are using custom bootstraps, then they know how to extend a pimple service. If they don't then they shouldn't be using a custom bootstrap.